src/store/getters.js

```diff
@@ -29,6 +29,7 @@ export default {
         type,
         ...acc,
         ...(type === ACCOUNT_HD_WALLET ? getHdWalletAccount(getters.wallet, idx) : {}),
+        protocol: 'aeternity',
```

Comment:
Not really needed. Chain type can be derived from the account address.
Aeternity: `ae_....` 
Bitcoin: `bc1......`
Dogecoin: `D....`

Reply:
I think it's better to have the `protocol` property instead of calculating it every time we need to know the exact protocol. Here's me comment on this: https://github.com/aeternity/superhero-wallet/pull/2183/files#r1276211607

Reply:
you can define a `protocol` getter based on address, so it would behave like a property 🙃

Reply:
I have completely removed the `deriveProtocolFromAddress` method as it is a must to know the protocol upfront to derive the account from seed to use the appropriate derivation path.